po/R-zh_CN.po

```diff
@@ -87,87 +87,95 @@ msgid "; filled with NA"
 msgstr ""
 
 msgid "A column may not be called .SD. That has special meaning."
-msgstr ""
+msgstr "无法将列命名为 .SD，因为 .SD 为特殊符号。"
 
 msgid "class must be length 1"
-msgstr ""
+msgstr "class 的长度必须为 1"
 
 msgid "between has been passed an argument x of type logical"
-msgstr ""
+msgstr "传入 between 的参数 x 为逻辑（logical）型"
 
 msgid ""
 "'between' function the 'x' argument is a POSIX class while 'lower' was not, "
 "coercion to POSIX failed with:"
 msgstr ""
+"'between' 中的 'x' 参数为 POSIX 类，而 'lower' 并不是，"
+"将 'lower' 强制转换成 POSIX 失败："
 
 msgid ""
 "'between' function the 'x' argument is a POSIX class while 'upper' was not, "
 "coercion to POSIX failed with:"
 msgstr ""
+"'between' 中的 'x' 参数为 POSIX 类，而 'upper' 并不是，"
+"将 'upper' 强制转换成 POSIX 失败："
 
 msgid ""
 "'between' lower= and upper= are both POSIXct but have different tzone "
 "attributes:"
 msgstr ""
+"'between' 中的 lower= 和 upper= 均为 POSIXct 类型但却有不同的时区"
+"属性（tzone）："
 
 msgid ". Please align their time zones."
-msgstr ""
+msgstr "。请确保二者的时区一致。"
 
 msgid ""
 "'between' arguments are all POSIXct but have mismatched tzone attributes:"
 msgstr ""
+"'between' 的参数均为 POSIXct 类型但时区属性（tzone）不匹配："
 
 msgid ". The UTC times will be compared."
-msgstr ""
+msgstr "。将采用 UTC 时间进行比较。"
 
 msgid "trying to use integer64 class when 'bit64' package is not installed"
-msgstr ""
+msgstr "试图使用 intger64 类型但 'bit64' 包尚未安装"
 
 msgid ""
 "Not yet implemented NAbounds=TRUE for this non-numeric and non-character type"
 msgstr ""
+"将 NAbounds=TRUE 应用于非数值（numeric）型和非字符串（character）型的功能尚未开发"
 
 msgid "Some lower>upper for this non-numeric and non-character type"
-msgstr ""
+msgstr "对于该非数值（numeric）型和非字符串（character）型的输入，存在一部分下界（lower）> 上界（upper）的情况"
 
 msgid "RHS has length()"
-msgstr ""
+msgstr "右手侧（RHS）的长度为"
 
 msgid "; expecting length 2."
-msgstr ""
+msgstr "；其长度应为 2。"
 
 msgid "c"
 msgstr ""
```

Comment:
if there is no translation needed, do we need to repeat msg.id in msg.str? Like

msgid "c"
msgstr "c"

Reply:
no, if `msgstr` is empty, simply no translation is done. Some people have been including the duplicate translation, others not... I don't have strong feelings one way or the other